frontend/src/views/Environments/EnvironmentCreateForm.js

```diff
@@ -285,6 +296,42 @@ const EnvironmentCreateForm = (props) => {
                   {`cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess aws://ACCOUNT_ID/REGION`}
```

Comment:
Maybe we can remove this command from here as we are specifying another command to bootstrap in lines 331-333

Reply:
Okeeey, I see you want to leave options!